packages/types/src/external/vue.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Vue class prop type
+ *
+ * Vue seems to just use any behind the scenes
+ *
+ * @see https://vuejs.org/guide/essentials/class-and-style.html#class-and-style-bindings
+ */
+export type VueClassProp =
```

Comment:
It can also be a `MaybeComputedOrRef`

Reply:
This would add vue as a dependency so either we leave it out, or just move this type into the client. Maybe I'll move it into the client as thats becoming our "core" now